demo/clearbutton_component/run.py

```diff
@@ -0,0 +1,12 @@
+import gradio as gr
+
+css = (
+    "footer {display: none !important;} .gradio-container {min-height: 0px !important;}"
+)
```

Comment:
We shouldn't use custom css on any of these component demos since the Spaces frame removes the footer anyways

Reply:
I think this is for the gradio footer (`Show Api` link) which does not get removed on spaces.

Reply:
Sorry no I meant embedding using web components (like we do on the docs page) already removes the footer, so we don't need to do this extra css

Reply:
Agree with @abidlabs, we have that css in most/all component demos because that was before embedding with web components

Reply:
Oh nice - will remove then!